.github/workflows/test.yml

```diff
@@ -24,13 +25,15 @@ jobs:
           - gemfiles/rails5.2.gemfile
           - gemfiles/rails6.0.gemfile
           - gemfiles/rails6.1.gemfile
+        include:
+          - {ruby: '3.1', gemfile: gemfiles/rails7.0.gemfile, experimental: true}
         exclude:
-          - {ruby: '3.0', gemfile: gemfiles/rails5.1.gemfile}
-          - {ruby: '3.0', gemfile: gemfiles/rails5.2.gemfile}
-          - {ruby: '3.1', gemfile: gemfiles/rails5.1.gemfile}
-          - {ruby: '3.1', gemfile: gemfiles/rails5.2.gemfile}
-          - {ruby: '3.2', gemfile: gemfiles/rails5.1.gemfile}
-          - {ruby: '3.2', gemfile: gemfiles/rails5.2.gemfile}          
+          - {ruby: '3.0', gemfile: gemfiles/rails5.1.gemfile, experimental: false}
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixexclude says that

> An excluded configuration only has to be a partial match for it to be excluded.

so I don’t think the `experimental: false` is necessary.